docs/docs/codes/modifiers.mdx

```diff
@@ -30,13 +30,15 @@ These act like any other key code.
 
 ### Modifier Functions
 
-These functions take the form: `XX(code)`
-
-- They _apply_ the modifier to another _code_.
-  - e.g. `&kp LS(A)` = `LEFT_SHIFT`+`A`
-    - aka a _shifted_ (capitalized) **A**.
-  - The _code_ and the _modifier_ are handled _as one_.
-- They can be _combined_:
-  - e.g. `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
-- Some codes _include_ modifiers in their definition.
-  - e.g. `DOLLAR` = `LS(N4)`
+Modifier functions add one or more modifier to another code. These functions take the form: `XX(code)`
```

Comment:
> Modifier functions add one or more modifier to another code.  These functions take the form: `XX(code)`

Recommend a new line between those sentences for readability.

Reply:
ok